index.html

```diff
@@ -2758,10 +2758,22 @@ <h3>Terms of Use</h3>
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://www.w3.org/2018/credentials/examples/v1"
+    "https://www.w3.org/2018/credentials/examples/v1",
+    {
+        "VerifiablePresentationTermsOfUseExtension": {
```

Comment:
```suggestion
        "@protected": true,
        "VerifiablePresentationTermsOfUseExtension": {
```

Reply:
Thanks for this addition. I thought we could intentionally drop this one since the term is protected (and only a single term is being defined in here. I'll go ahead and add this in, but for my knowledge what's the reason we need it listed in both places?

Reply:
The `@protected` flag below applies to the scoped context within the `VerifiablePresentationTermsOfUseExtension` term definition, thereby protecting all terms in *that* context. The one above applies to the context that defines `VerifiablePresentationTermsOfUseExtension`, protecting all terms in it. IOW, there are two contexts here -- and each one should set `@protected` to protect all terms in those contexts.